### ISSUE TITLE
Unable to use dotnet new to create a project

### ISSUE THREAD

**Issue by mjmilan** [dotnet/cli#4274]
Hi folks,

I am trying to get a basic application working under dotnet core... We are working on a goal to double the number of requests we can handle in the next three years following a re-write, and basing our backend on dotnet core is therefore very attractive in terms of the performance of your server itself (Kestrel?)...

I want to push this hard with our architects, but if we can't even get a successful starter project going then I can look forward to getting shot down in flames!
Martin.
## Steps to reproduce

Go into an empty folder via console
type "dotnet new MyApp" and hit enter
## Expected  behavior

I expected the system to scaffold me a new application together that I can actually get on with using...
## Actual behavior

Seems the dotnet program encounters an exception whilst installing:

E:\Sandbox\WebApplication1>dotnet new MyApp
## Welcome to .NET Core!

Learn more about .NET Core @ https://aka.ms/dotnet-docs. Use dotnet --help to see available commands or go to https://ak
a.ms/dotnet-cli-docs.
## Telemetry

The .NET Core tools collect usage data in order to improve your experience. The data is anonymous and does not include c
ommandline arguments. The data is collected by Microsoft and shared with the community.
You can opt out of telemetry by setting a DOTNET_CLI_TELEMETRY_OPTOUT environment variable to 1 using your favorite shel
l.
You can read more about .NET Core tools telemetry @ https://aka.ms/dotnet-cli-telemetry.
## Configuring...

A command is running to initially populate your local package cache, to improve restore speed and enable offline access.
 This command will take up to a minute to complete and will only happen once.
Decompressing 100% 2361 ms
Expanding 24%
Unhandled Exception: System.AggregateException: One or more errors occurred. (One or more errors occurred. (Access to th
e path 'C:\Users\martinm\AppData\Local\Temp\ycywbvye.fjf\Microsoft.CodeAnalysis.Common\1.3.0\ThirdPartyNotices.rtf' is d
enied.)) (Access to the path 'C:\Users\martinm\AppData\Local\Temp\ycywbvye.fjf\Microsoft.CodeAnalysis.Common\1.3.0\Third
PartyNotices.rtf' is denied.) ---> System.UnauthorizedAccessException: Access to the path 'C:\Users\martinm\AppData\Loca
l\Temp\ycywbvye.fjf\Microsoft.CodeAnalysis.Common\1.3.0\ThirdPartyNotices.rtf' is denied.
   at System.IO.Win32FileStream..ctor(String path, FileMode mode, FileAccess access, FileShare share, Int32 bufferSize,
FileOptions options, FileStream parent)
   at System.IO.Win32FileSystem.Open(String fullPath, FileMode mode, FileAccess access, FileShare share, Int32 bufferSiz
e, FileOptions options, FileStream parent)
   at System.IO.FileStream.Init(String path, FileMode mode, FileAccess access, FileShare share, Int32 bufferSize, FileOp
tions options)
   at System.IO.File.Create(String path)
   at Microsoft.DotNet.Archive.IndexedArchive.CopyOperation.Execute()
   at Microsoft.DotNet.Archive.IndexedArchive.ExtractOperation.DoOperation()
   at Microsoft.DotNet.Archive.IndexedArchive.<>c__DisplayClass19_1.<Extract>b__0(ExtractOperation extractOperation)
   at System.Linq.Parallel.ForAllOperator`1.ForAllEnumerator`1.MoveNext(TInput& currentElement, Int32& currentKey)
   at System.Linq.Parallel.ForAllSpoolingTask`2.SpoolingWork()
   at System.Linq.Parallel.SpoolingTaskBase.Work()
   at System.Linq.Parallel.QueryTask.BaseWork(Object unused)
   at System.Linq.Parallel.QueryTask.<>c.<.cctor>b__10_0(Object o)
   at System.Threading.Tasks.Task.Execute()
   --- End of inner exception stack trace ---
   at System.Linq.Parallel.QueryTaskGroupState.QueryEnd(Boolean userInitiatedDispose)
   at System.Linq.Parallel.MergeExecutor`1.Execute()
   at System.Linq.Parallel.MergeExecutor`1.Execute[TKey](PartitionedStream`2 partitions, Boolean ignoreOutput, ParallelM
ergeOptions options, TaskScheduler taskScheduler, Boolean isOrdered, CancellationState cancellationState, Int32 queryId)

   at System.Linq.Parallel.PartitionedStreamMerger`1.Receive[TKey](PartitionedStream`2 partitionedStream)
   at System.Linq.Parallel.ForAllOperator`1.WrapPartitionedStream[TKey](PartitionedStream`2 inputStream, IPartitionedStr
eamRecipient`1 recipient, Boolean preferStriping, QuerySettings settings)
   at System.Linq.Parallel.UnaryQueryOperator`2.UnaryQueryOperatorResults.ChildResultsRecipient.Receive[TKey](Partitione
dStream`2 inputStream)
   at System.Linq.Parallel.ListQueryResults`1.GivePartitionedStream(IPartitionedStreamRecipient`1 recipient)
   at System.Linq.Parallel.UnaryQueryOperator`2.UnaryQueryOperatorResults.GivePartitionedStream(IPartitionedStreamRecipi
ent`1 recipient)
   at System.Linq.Parallel.QueryOperator`1.GetOpenedEnumerator(Nullable`1 mergeOptions, Boolean suppressOrder, Boolean f
orEffect, QuerySettings querySettings)
   at System.Linq.Parallel.ForAllOperator`1.RunSynchronously()
   at System.Linq.ParallelEnumerable.ForAll[TSource](ParallelQuery`1 source, Action`1 action)
   at Microsoft.DotNet.Archive.IndexedArchive.Extract(String compressedArchivePath, String outputDirectory, IProgress`1
progress)
   at Microsoft.DotNet.Configurer.NuGetPackagesArchiver.ExtractArchive()
   at Microsoft.DotNet.Configurer.NuGetCachePrimer.PrimeCache()
   at Microsoft.DotNet.Configurer.DotnetFirstTimeUseConfigurer.Configure()
   at Microsoft.DotNet.Cli.Program.ConfigureDotNetForFirstTimeUse(INuGetCacheSentinel nugetCacheSentinel)
   at Microsoft.DotNet.Cli.Program.ProcessArgs(String[] args, ITelemetry telemetryClient)
   at Microsoft.DotNet.Cli.Program.Main(String[] args)

I've also tried to create a new project using VS2015 - here is the log output...

C:\Program Files\dotnet\dotnet.exe restore "E:\Sandbox\WebApplication2.vs\restore.dg" -s "C:\Program Files (x86)\Microsoft SDKs\NuGetPackages"
log  : Restoring packages for E:\Sandbox\WebApplication2\src\WebApplication2\project.json...
log  : Installing System.Runtime.Handles 4.0.0.
log  : Installing Microsoft.AspNetCore.Razor 1.0.0.
log  : Installing Microsoft.DotNet.InternalAbstractions 1.0.0.
log  : Installing Microsoft.AspNetCore.Routing.Abstractions 1.0.0.
log  : Installing Microsoft.AspNetCore.DataProtection.Abstractions 1.0.0.
log  : Installing Microsoft.AspNetCore.Cryptography.Internal 1.0.0.
log  : Installing System.IO.FileSystem.Primitives 4.0.0.
log  : Installing System.IO.FileSystem 4.0.0.
log  : Installing System.Runtime.InteropServices 4.0.20.
log  : Installing System.Xml.XPath 4.0.1.
log  : Installing System.ComponentModel.Primitives 4.1.0.
log  : Installing System.Collections.Specialized 4.0.1.
log  : Installing System.Collections.NonGeneric 4.0.1.
log  : Installing System.Diagnostics.Contracts 4.0.1.
log  : Installing Microsoft.AspNetCore.Razor.Runtime 1.0.0.
log  : Installing Microsoft.Extensions.Globalization.CultureInfoCache 1.0.0.
log  : Installing Microsoft.NETCore.App 1.0.1.
log  : Installing Microsoft.ApplicationInsights.AspNetCore 1.0.0.
log  : Installing Microsoft.AspNetCore.Mvc 1.0.1.
log  : Installing Microsoft.Extensions.Localization.Abstractions 1.0.0.
log  : Installing Microsoft.AspNetCore.Routing 1.0.1.
log  : Installing Microsoft.AspNetCore.Mvc.Abstractions 1.0.1.
log  : Installing Microsoft.AspNetCore.Server.IISIntegration 1.0.0.
log  : Installing Microsoft.AspNetCore.Server.Kestrel 1.0.1.
log  : Installing Microsoft.Extensions.DependencyModel 1.0.0.
log  : Installing Microsoft.AspNetCore.Mvc.Abstractions 1.0.0.
log  : Installing Microsoft.Extensions.Configuration.EnvironmentVariables 1.0.0.
log  : Installing Microsoft.Extensions.Configuration.FileExtensions 1.0.0.
log  : Installing Microsoft.Extensions.Configuration.Json 1.0.0.
log  : Installing Microsoft.AspNetCore.Authorization 1.0.0.
log  : Installing Microsoft.Extensions.Logging 1.0.0.
log  : Installing Microsoft.Extensions.Logging.Console 1.0.0.
log  : Installing Microsoft.AspNetCore.DataProtection 1.0.0.
log  : Installing Microsoft.Extensions.Logging.Debug 1.0.0.
log  : Installing Microsoft.Extensions.Options.ConfigurationExtensions 1.0.0.
log  : Installing Libuv 1.9.0.
log  : Installing Microsoft.CodeAnalysis.CSharp 1.3.0.
log  : Installing Microsoft.CodeAnalysis.VisualBasic 1.3.0.
log  : Installing Microsoft.CSharp 4.0.1.
log  : Installing Microsoft.NETCore.DotNetHostPolicy 1.0.1.
log  : Installing Microsoft.NETCore.Runtime.CoreCLR 1.0.4.
log  : Installing System.Net.WebSockets 4.0.0.
log  : Installing Microsoft.VisualBasic 10.0.1.
log  : Installing NETStandard.Library 1.6.0.
log  : Installing System.Buffers 4.0.0.
log  : Installing System.Collections.Immutable 1.2.0.
log  : Installing System.ComponentModel 4.0.1.
log  : Installing System.ComponentModel.Annotations 4.1.0.
log  : Installing System.Diagnostics.DiagnosticSource 4.0.0.
log  : Installing System.Xml.ReaderWriter 4.0.10.
log  : Installing System.Reflection.Primitives 4.0.0.
log  : Installing System.Diagnostics.Process 4.1.0.
log  : Installing System.Dynamic.Runtime 4.0.11.
log  : Installing System.Globalization.Extensions 4.0.1.
log  : Installing System.IO.FileSystem.Watcher 4.0.0.
log  : Installing System.IO.MemoryMappedFiles 4.0.0.
log  : Installing System.IO.UnmanagedMemoryStream 4.0.1.
log  : Installing System.Linq.Expressions 4.1.0.
log  : Installing System.Net.Primitives 4.0.0.
log  : Installing System.Linq.Parallel 4.0.1.
log  : Installing System.Linq.Queryable 4.0.1.
log  : Installing System.Net.NameResolution 4.0.0.
log  : Installing System.Net.Requests 4.0.11.
log  : Installing System.Runtime.InteropServices 4.0.0.
log  : Installing System.Net.Security 4.0.0.
log  : Installing System.Net.WebHeaderCollection 4.0.1.
log  : Installing System.Numerics.Vectors 4.1.1.
log  : Installing System.Reflection.DispatchProxy 4.0.1.
log  : Installing System.Reflection.Metadata 1.3.0.
log  : Installing System.Reflection.TypeExtensions 4.1.0.
log  : Installing System.Resources.ResourceManager 4.0.0.
log  : Installing System.Resources.Reader 4.0.0.
log  : Installing System.Runtime.Loader 4.0.0.
log  : Installing System.Security.Cryptography.Algorithms 4.2.0.
log  : Installing System.Security.Cryptography.Encoding 4.0.0.
log  : Installing System.Security.Cryptography.Primitives 4.0.0.
log  : Installing System.Security.Cryptography.X509Certificates 4.1.0.
log  : Installing runtime.native.System.IO.Compression 4.1.0.
log  : Installing System.Threading.Tasks.Dataflow 4.6.0.
log  : Installing Microsoft.NETCore.DotNetHost 1.0.1.
log  : Installing System.Xml.XPath.XDocument 4.0.1.
log  : Installing System.Threading.Tasks.Extensions 4.0.0.
log  : Installing System.Threading.Tasks.Parallel 4.0.1.
log  : Installing System.Threading.Thread 4.0.0.
log  : Installing System.Threading.ThreadPool 4.0.10.
log  : Installing Microsoft.ApplicationInsights 2.1.0.
log  : Installing System.Xml.XmlDocument 4.0.1.
log  : Installing Microsoft.AspNetCore.Hosting.Abstractions 1.0.0.
log  : Installing Microsoft.AspNetCore.Http.Abstractions 1.0.0.
log  : Installing Microsoft.AspNetCore.Mvc.ViewFeatures 1.0.0.
log  : Installing Microsoft.Extensions.Configuration 1.0.0.
log  : Installing System.Text.Encoding.CodePages 4.0.1.
log  : Installing Microsoft.Extensions.DiagnosticAdapter 1.0.0.
log  : Installing System.Diagnostics.FileVersionInfo 4.0.0.
log  : Installing Microsoft.Extensions.Logging.Abstractions 1.0.0.
log  : Installing Microsoft.AspNetCore.Mvc.ApiExplorer 1.0.1.
log  : Installing Microsoft.CodeAnalysis.Analyzers 1.1.0.
log  : Installing Microsoft.AspNetCore.Mvc.Cors 1.0.1.
log  : Installing Microsoft.AspNetCore.Mvc.DataAnnotations 1.0.1.
log  : Installing Microsoft.AspNetCore.Mvc.Formatters.Json 1.0.1.
log  : Installing Microsoft.AspNetCore.Mvc.Localization 1.0.1.
log  : Installing Microsoft.AspNetCore.Mvc.Razor 1.0.1.
log  : Installing Microsoft.AspNetCore.Mvc.TagHelpers 1.0.1.
log  : Installing Microsoft.AspNetCore.Mvc.ViewFeatures 1.0.1.
log  : Installing Microsoft.Extensions.Caching.Memory 1.0.0.
log  : Installing Microsoft.Extensions.DependencyInjection 1.0.0.
log  : Installing System.Collections 4.0.11.
log  : Installing System.Text.RegularExpressions 4.1.0.
log  : Installing Microsoft.AspNetCore.Http.Extensions 1.0.0.
log  : Installing Microsoft.AspNetCore.Routing.Abstractions 1.0.1.
log  : Installing Microsoft.Extensions.ObjectPool 1.0.0.
log  : Installing Microsoft.Extensions.Options 1.0.0.
log  : Installing System.Security.Principal.Windows 4.0.0.
log  : Installing Microsoft.AspNetCore.Http 1.0.0.
log  : Installing Microsoft.AspNetCore.HttpOverrides 1.0.0.
log  : Installing System.Diagnostics.Debug 4.0.11.
log  : Installing System.Globalization 4.0.11.
log  : Installing System.IO 4.1.0.
log  : Installing System.Linq 4.1.0.
log  : Installing System.Net.Primitives 4.0.11.
log  : Installing System.ComponentModel.TypeConverter 4.1.0.
log  : Installing System.Runtime.Extensions 4.1.0.
log  : Installing System.Runtime.InteropServices 4.1.0.
log  : Installing System.Text.Encoding 4.0.11.
log  : Installing System.Threading 4.0.11.
log  : Installing Microsoft.Extensions.PlatformAbstractions 1.0.0.
log  : Installing System.Threading.Tasks 4.0.11.
log  : Installing System.Threading.Timer 4.0.1.
log  : Installing System.Diagnostics.StackTrace 4.0.1.
log  : Installing Microsoft.AspNetCore.Hosting 1.0.0.
log  : Installing System.Runtime.InteropServices.RuntimeInformation 4.0.0.
log  : Installing Microsoft.AspNetCore.WebUtilities 1.0.0.
log  : Installing System.AppContext 4.1.0.
log  : Installing Microsoft.Extensions.FileProviders.Physical 1.0.0.
log  : Installing System.IO.FileSystem 4.0.1.
log  : Installing System.Runtime.Serialization.Primitives 4.1.1.
log  : Installing Newtonsoft.Json 9.0.1.
log  : Installing Microsoft.Extensions.DependencyInjection.Abstractions 1.0.0.
log  : Installing System.Console 4.0.0.
log  : Installing Microsoft.Extensions.Configuration.Abstractions 1.0.0.
log  : Installing Microsoft.Extensions.Configuration.Binder 1.0.0.
log  : Installing Microsoft.NETCore.Platforms 1.0.1.
log  : Installing Microsoft.CodeAnalysis.Common 1.3.0.
log  : Installing System.ObjectModel 4.0.12.
log  : Installing System.Reflection 4.1.0.
log  : Installing System.Reflection.Extensions 4.0.1.
log  : Installing Microsoft.Net.Http.Headers 1.0.0.
log  : Installing System.Reflection.Primitives 4.0.1.
log  : Installing System.Resources.ResourceManager 4.0.1.
log  : Installing Microsoft.NETCore.Targets 1.0.1.
log  : Installing Microsoft.Extensions.Caching.Abstractions 1.0.0.
log  : Installing System.Runtime 4.1.0.
log  : Installing Microsoft.NETCore.DotNetHostResolver 1.0.1.
log  : Installing Microsoft.AspNetCore.Antiforgery 1.0.1.
log  : Installing Microsoft.NETCore.Jit 1.0.4.
log  : Installing Microsoft.NETCore.Windows.ApiSets 1.0.1.
log  : Installing Microsoft.Win32.Primitives 4.0.1.
log  : Installing System.Collections.Concurrent 4.0.12.
log  : Installing System.Diagnostics.Tools 4.0.1.
log  : Installing System.Diagnostics.Tracing 4.1.0.
log  : Installing System.Globalization.Calendars 4.0.1.
log  : Installing System.IO.Compression 4.1.0.
log  : Installing Microsoft.Extensions.Primitives 1.0.0.
log  : Installing Microsoft.Extensions.FileSystemGlobbing 1.0.0.
log  : Installing System.IO.Compression.ZipFile 4.0.1.
log  : Installing Microsoft.Extensions.FileProviders.Composite 1.0.0.
log  : Installing System.IO.FileSystem.Primitives 4.0.1.
log  : Installing Microsoft.AspNetCore.Mvc.Razor.Host 1.0.1.
log  : Installing System.Net.Http 4.1.0.
log  : Installing System.Net.Sockets 4.1.0.
log  : Installing Microsoft.AspNetCore.Localization 1.0.0.
log  : Installing Microsoft.AspNetCore.JsonPatch 1.0.0.
log  : Installing System.Runtime.Handles 4.0.1.
log  : Installing System.Runtime.Numerics 4.0.1.
log  : Installing System.Text.Encoding.Extensions 4.0.11.
log  : Installing System.Xml.ReaderWriter 4.0.11.
log  : Installing System.Xml.XDocument 4.0.11.
log  : Installing Microsoft.Extensions.Localization 1.0.0.
log  : Installing Microsoft.Win32.Registry 4.0.0.
log  : Installing runtime.native.System 4.0.0.
log  : Installing System.Reflection.Emit 4.0.1.
log  : Installing System.Reflection.Emit.ILGeneration 4.0.1.
log  : Installing System.Threading.Overlapped 4.0.1.
log  : Installing System.Reflection.Emit.Lightweight 4.0.1.
log  : Installing runtime.native.System.Net.Security 4.0.1.
log  : Installing runtime.native.System.Security.Cryptography 4.0.0.
log  : Installing System.Security.Claims 4.0.1.
log  : Installing System.Security.Cryptography.OpenSsl 4.0.0.
log  : Installing System.Security.Principal 4.0.1.
log  : Installing Microsoft.AspNetCore.Cors 1.0.0.
log  : Installing runtime.native.System.Net.Http 4.0.1.
log  : Installing Microsoft.AspNetCore.Mvc.Core 1.0.1.
log  : Installing System.Security.Cryptography.Cng 4.2.0.
log  : Installing System.Security.Cryptography.Csp 4.0.0.
log  : Installing System.Collections 4.0.10.
log  : Installing System.Diagnostics.Debug 4.0.10.
log  : Installing System.Diagnostics.Tools 4.0.0.
log  : Installing System.Diagnostics.Tracing 4.0.20.
log  : Installing System.Globalization 4.0.10.
log  : Installing System.IO 4.0.10.
log  : Installing System.IO.Compression 4.0.0.
log  : Installing System.Linq 4.0.0.
log  : Installing System.Net.Http 4.0.0.
log  : Installing System.Reflection 4.0.10.
log  : Installing Microsoft.Extensions.WebEncoders 1.0.0.
log  : Installing System.Reflection.Extensions 4.0.0.
log  : Installing System.Runtime 4.0.20.
log  : Installing System.Runtime.Extensions 4.0.10.
log  : Installing System.Text.Encoding 4.0.10.
log  : Installing System.Text.Encoding.Extensions 4.0.10.
log  : Installing System.Text.RegularExpressions 4.0.10.
log  : Installing System.Threading 4.0.10.
log  : Installing System.Threading.Tasks 4.0.10.
log  : Installing System.Xml.XDocument 4.0.10.
log  : Installing Microsoft.AspNetCore.Hosting.Server.Abstractions 1.0.0.
log  : Installing Microsoft.Extensions.FileProviders.Abstractions 1.0.0.
log  : Installing Microsoft.AspNetCore.Http.Features 1.0.0.
log  : Installing System.Text.Encodings.Web 4.0.0.
log  : Installing Microsoft.AspNetCore.Antiforgery 1.0.0.
log  : Installing Microsoft.AspNetCore.Diagnostics.Abstractions 1.0.0.
log  : Installing Microsoft.AspNetCore.Html.Abstractions 1.0.0.
log  : Installing Microsoft.AspNetCore.Mvc.Core 1.0.0.
log  : Installing Microsoft.AspNetCore.Mvc.DataAnnotations 1.0.0.
log  : Installing Microsoft.AspNetCore.Mvc.Formatters.Json 1.0.0.
error: Access to the path 'C:\Users\martinm.nuget\packages\Microsoft.CodeAnalysis.Common\1.3.0\ThirdPartyNotices.rtf' is denied.

Please note I have tried clearing my packages folder before running dotnet new - no effect - same issues...
## Environment data

`dotnet --info` output:

.NET Command Line Tools (1.0.0-preview2-003131)

Product Information:
 Version:            1.0.0-preview2-003131
 Commit SHA-1 hash:  635cf40e58

Runtime Environment:
 OS Name:     Windows
 OS Version:  6.1.7601
 OS Platform: Windows
 RID:         win7-x64

**Comment by Nick-Lucas** [comment]
This makes it look like either a permissions issue, or maybe something like a special character in your username causing an issue  with the folder structure `'C:\Users\martinm.nuget\packages\Microsoft.CodeAnalysis.Common\1.3.0\ThirdPartyNotices.rtf' is denied.`

I would expect to see `'martinm\.nuget` not `'martinm.nuget`

Perhaps try a [takeown[(https://technet.microsoft.com/en-us/library/cc753024(v=ws.11).aspx) of `C:\Users\martinm\.nuget`?

**Comment by mjmilan** [comment]
I agree with the "martinm.nuget" looking out of place - I noticed that myself... I'll try your takeown idea in the morning and let you know how that goes - I'm at home now and quite deliberately powered down my office machine before I left...

One of those days.

My username on the system is simply martinm...

Cheers for taking the time...

**Comment by mjmilan** [comment]
Having applied the takeown command as suggested, the fault remains...

**Comment by mjmilan** [comment]
Interestingly, I did manage to get a working installation on my personal laptop last night - Windows10...

**Comment by Nick-Lucas** [comment]
Interesting, well when I've seen things like missing '/' characters before, it's usually because something far further up the process has failed and the crash is a symptom of something deeper, but it was good to rule that out first.

What I can say is the fantastic nice thing about dotnet CLI is that it just works without any other setup or configuration. This might not sound that helpful, but it leads me to believe your installation may have been corrupted at some stage, or worse there's something up with your OS installation.

I'd go for re-install of dotnet, making sure you're on an admin account and with manually elevated privileges for the installer. If that doesn't work I really wouldn't know where to go, but I'm not an expert in dotnet's inner workings or what logs it keeps that could be helpful

**Comment by mjmilan** [comment]
Hi again Nick...

Just tried running this through having uninstalled and reinstalled (both as administrator) - still no joy... Same issue...

**Comment by piotrpMSFT** [comment]
@mjmilan do either of the problematic paths actually point to a file on your system? Where is your nuget package cache located? Are there any nuget.config files above your project which might be getting in the way?

As a potential workaround, the CLI build uses a custom nuget package cache location to handle several test scenarios for restore: https://github.com/dotnet/cli/blob/88a1bdd63957aa7100ca89fe1bc3bf2c424985fd/run-build.ps1#L32

you could try setting a custom package cache location before running restore to see if this clears up the issue.

I offer a workaround as this seems like a local machine configuration issue, off the cuff. The questions above might help us track the issue down, but I'd like to get you unblocked :)

**Comment by mjmilan** [comment]
Thanks for the suggestions...

I am travelling into the office at the moment, so I'll give it a whirl
later this morning...

**Comment by piotrpMSFT** [comment]
@mjmilan I'm going to close the issue since it's been idle for quite some time. Please open a new issue if needed.
